pkg/scannerv4/client/client.go

```diff
@@ -53,7 +53,7 @@ type Scanner interface {
 	GetMatcherMetadata(context.Context) (*v4.Metadata, error)
 
 	// GetSBOM to get sbom for an image
-	GetSBOM(ctx context.Context, name string, ref name.Digest) ([]byte, bool, error)
+	GetSBOM(ctx context.Context, name string, ref name.Digest, namespace string) ([]byte, bool, error)
```

Comment:
just curious why the change between `namespace` and `uri`?

Reply:
Woops, that was left over, should be `uri` now.

Thought process was although for SPDX this value will used in the document namespace field, in other specs it may be called something else - `uri` seemed to be the most accurate for 'what' we're putting in the field vs. assuming how it will be used. (may be flawed thing).

Reply:
https://github.com/stackrox/stackrox/pull/14135/commits/9a56f2c639556e44a5e88be7f9475996b12b2fd7